src/typing_extensions.py

```diff
@@ -413,12 +413,12 @@ def clear_overloads():
 OrderedDict = typing.OrderedDict
 Counter = typing.Counter
 ChainMap = typing.ChainMap
-Text = typing.Text
+Text = typing.Text  # noqa: UP019
```

Comment:
Can we just turn off the UP* rules that tell you about deprecated parts of typing? I don't think they're helpful for this project.

Reply:
Fair point :) I'll make the change later